CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 * Update favicon to be the Elastic Package Registry logo. [#858](https://github.com/elastic/package-registry/pull/858)
-* Implement proxy mode. [#860](https://github.com/elastic/package-registry/pull/860) [#871](https://github.com/elastic/package-registry/pull/871)
+* Implement proxy mode. [#860](https://github.com/elastic/package-registry/pull/860) [#871](https://github.com/elastic/package-registry/pull/871) [#873](https://github.com/elastic/package-registry/pull/873)
```

Comment:
As there has not been any release with the proxy mode enabled , I've just added this PR here